webapp/channels/src/components/textbox.test.tsx

```diff
@@ -28,106 +26,118 @@ describe('components/TextBox', () => {
         },
         useChannelMentions: true,
         tabIndex: 0,
+        id: '',
+        value: '',
+        onChange: jest.fn(),
+        onKeyPress: jest.fn(),
+        createMessage: '',
+        characterLimit: 0,
     };
 
     test('should match snapshot with required props', () => {
-        function emptyFunction() {} //eslint-disable-line no-empty-function
+        const emptyFunction = jest.fn();
+        const props = {
+            ...baseProps,
+            id: 'someid',
+            value: 'some test text',
+            onChange: emptyFunction,
+            onKeyPress: emptyFunction,
+            characterLimit: 400,
+            createMessage: 'placeholder text',
+            supportsCommands: false,
+        };
 
         const wrapper = shallow(
-            <Textbox
-                id='someid'
-                value='some test text'
-                onChange={emptyFunction}
-                onKeyPress={emptyFunction}
-                characterLimit={4000}
-                createMessage='placeholder text'
-                supportsCommands={false}
-                {...baseProps}
-            />,
+            <Textbox {...props}/>,
         );
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot with additional, optional props', () => {
-        function emptyFunction() {} //eslint-disable-line no-empty-function
+        const emptyFunction = jest.fn();
+        const props = {
+            ...baseProps,
+            id: 'someid',
+            value: 'some test text',
+            onChange: emptyFunction,
+            onKeyPress: emptyFunction,
+            characterLimit: 4000,
+            createMessage: 'placeholder text',
+            supportsCommands: false,
+            rootId: 'root_id',
+            onComposition: jest.fn().mockReturnValue({}),
```

Comment:
This is technically changing the behaviour of these tests slightly since these functions now return an empty object instead of nothing, but since the tests aren't breaking, that's not a problem

Reply:
we will run e2e to be sure we didnt break any of them